android/src/main/java/com/reactnativechangeicon/ChangeIconModule.java

```diff
@@ -91,6 +92,7 @@ public void changeIcon(String iconName, Promise promise) {
         this.componentClass = activeClass;
         activity.getApplication().registerActivityLifecycleCallbacks(this);
         iconChanged = true;
+        this.completeIconChange();
```

Comment:
Please add a comment why is this required

Reply:
@abhishekshori1 Addressed.